src/gameboy/include/gameboy/gameboy.h

```diff
@@ -34,6 +35,15 @@ using namespace controller;
 
 namespace gameboy {
 
+/**
+ * Specifies all the Debug configurations that should go into GameBoy
+ */
+struct DebugOptions {
+	std::string rom_path;
```

Comment:
rom_path is not a debug option, it needs to be passed regardless of Debugging.

Move this back out as a normal parameter.